s5/s5.go

```diff
@@ -57,6 +57,10 @@ func (s *S5) Run() error {
 		return err
 	}
 
+	if ! sqlReader.HasData() {
+		return nil
```

Comment:
I'm thinking if the correct behaviour encountering an empty table is to create an empty file on S3. Empty table is in a valid state so the valid counter part would be an empty file.